src/commands/to-destructuring-assignment.md

```diff
@@ -0,0 +1,56 @@
+# `to-destructuring-assignment`
```

Comment:
```suggestion
# `to-destructure`
```

I think this would do already. So that we could also support shorthand up to `2dest`